config/initializers/feature_toggle.rb

```diff
@@ -20,4 +20,10 @@ def self.location_toggle_enabled?
 
     false
   end
+
+  def self.new_locations_table_layout_enabled?
```

Comment:
i know you're just copying old code (I did the same) but it occurs to me every time i see these they could just be 
```suggestion
  def self.new_locations_table_layout_enabled?
       !Rails.env.production?
  end
```